### PR TITLE
yaml: Add support for Zephyr OS

### PR DESCRIPTION
**Pull request by rshym**

### CODE REVIEW

File: prod-devel-rcar.yaml [hunk 1]
```diff
@@ -307,6 +310,22 @@ components:
       additional_deps:
         - "../android_kernel/out/android12-5.4/common/arch/arm64/boot/Image"
 
+  zephyr:
+    build-dir: "zephyr"
+    sources:
+      - type: west
+        # `url`, `rev` and `file` are optional items.
```
Comment (lorc) [inline prod-devel-rcar.yaml:317]: I don't think we need this. It is already covered either in Mouling or West documentation.

**Review by lorc — APPROVED** [review]
With that minor comment addressed:

`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

File: prod-devel-rcar.yaml (hunk 1 above)
Comment (rshym) [reply prod-devel-rcar.yaml:317]: Thanks, I removed the whole commented block. It looks cleaner.

**Comment by rshym** [comment]
Version is bumped to 0.8, according to a required change in moulin.
`board` field is set to `xenvm`.
`Ack` is added.Ready t

**Pull request merged**